src/styles.sass

```diff
@@ -932,5 +932,17 @@ details summary::marker
     width: fit-content
     font-size: 9px
 
+.subnav-search-icon 
+    position: absolute
+    top: 10px
+    left: 8px
+    display: block
+    text-align: center
+    pointer-events: none
+
+.highligth
```

Comment:
Spelling: `highlight`, not `highligth`, please check this everywhere

Reply:
Fixed